api/container/containerv1/properties.go

```diff
@@ -0,0 +1,53 @@
+package containerv1
+
+import (
+	"fmt"
+
+	"github.com/IBM-Cloud/bluemix-go/client"
+)
+
+//Vlan ...
+type DCVlan struct {
+	ID         string           `json:"id"`
+	Properties DCVlanProperties `json:"properties"`
+	VlanType   string           `json:"type"`
```

Comment:
Both type and vlan_type are referred as VlanType. Could you change the VlanType for type to type